src/Auth0.OidcClient.Core/Auth0ClientBase.cs

```diff
@@ -21,19 +21,26 @@ public abstract class Auth0ClientBase : IAuth0Client
         private readonly Auth0ClientOptions _options;
         private readonly string _userAgent;
         private IdentityModel.OidcClient.OidcClient _oidcClient;
+        private IdentityModel.OidcClient.OidcClient OidcClient
+        {
+            get
+            {
+                return _oidcClient ?? (_oidcClient = new IdentityModel.OidcClient.OidcClient(CreateOidcClientOptions(_options)));
+            }
+        }
```

Comment:
Just moved from below to keep this lazy instantiator with its private instance field.